doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -0,0 +1,162 @@
+build. There should be 17 of them at `<https://wheels.scipy.org>`_, 3 for Mac,
+6 for Windows, and 8 for Linux. Here the wheelhouse directory is in the home
+directory. Then upload the wheels. You will need to sign every file separately
+in the process::
+
+    $ cd ../terryfy
+    $ git pull origin master
+    $ CDN_URL=https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com
+    $ ./wheel-uploader -r warehouse -u $CDN_URL -s -v -w ~/wheelhouse -t win numpy 1.11.2rc1
+    $ ./wheel-uploader -r warehouse -u $CDN_URL -s -v -w ~/wheelhouse -t manylinux1 numpy 1.11.2rc1
+    $ ./wheel-uploader -r warehouse -u $CDN_URL -s -v -w ~/wheelhouse -t macosx numpy 1.11.2rc1
+
+If one of the commands breaks in the middle, you may need to manually upload
+the rest of the files using twine because wheel-uploader always starts from the
+beginning and PyPI does not allow the same file to be uploaded twice.
+
+The source files should be uploaded last to avoid problems::
+
+    $ cd ../numpy
+    $ twine upload -s release/installers/numpy-1.11.2rc1.zip
+    $ twine upload -s release/installers/numpy-1.11.2rc1.tar.gz
+
+If this is not a final release, log into PyPI and hide the new directory while
+making sure the last final release is visible.
+
+Upload tarballs to github
+-------------------------
+We want to post the hash values for the files uploaded to PyPI in the release
+generated README file, so first copy the wheels into the installers directory
+and remove the now unneeded signature files::
+
+    $ cp ~/wheelhouse/numpy-1.11.2rc1* release/installers
+    $ rm release/installers/*.asc
+
+Generate the ``release/Changelog`` and ``release/README`` files::
+
+    $ paver write_release_and_log
+
+Go to `<https://github.com/numpy/numpy/releases>`_, there should be a
+``v1.11.2rc1 tag``, hit the edit button for that tag. There are two ways to add
+files, using an editable text window and as binary uploads.
+
+- Cut-and-paste the ``release/README`` file contents into the text window.
+- Upload ``release/installers/numpy-1.11.2rc1.tar.gz`` as a binary file.
+- Upload ``release/installers/numpy-1.11.2rc1.zip`` as a binary file.
+- Upload ``release/README`` as a binary file.
+- Upload ``release/Changelog`` as a binary file (maybe).
+- Hit the ``Update release`` button at the bottom.
+
+
+Upload documents to docs.scipy.org
+----------------------------------
+
+This step is only needed for final releases and can be skipped for
+pre-releases. You will also need upload permission for the document server, if
+you do not have permission ping Pauli Virtanen or Ralf Gommers to generate and
+upload the documentation. Otherwise::
+
+    $ pushd doc
+    $ make dist
+    $ make upload USERNAME=<yourname> RELEASE=1.11.2rc1
+    $ popd
+
+If the release series is a new one, you will need to rebuild and upload the
+``docs.scipy.org`` front page::
+
+    $ cd ../docs.scipy.org
+    $ gvim index.rst 
```

Comment:
Send a PR to https://github.com/scipy/docs.scipy.org with the changes that add links to the new docs. See https://github.com/scipy/docs.scipy.org/pull/7 for an example